Project/Source/Components/UI/ComponentText.cpp

```diff
@@ -189,10 +190,12 @@ void ComponentText::RecalculcateVertices() {
 
 	float x = position.x;
 	float y = position.y;
-	// FontSize / size of imported font
-	float scale = fontSize / 48.0f;
 
-	for (int i = 0; i < text.size(); ++i) {	
+	float2 transformScale = transform->GetScale().xy();
+
+	float scale = (fontSize / 12) * (transformScale.x > transformScale.y ? transformScale.x : transformScale.y) * GetOwner().GetComponent<ComponentCanvasRenderer>()->GetCanvasScreenFactor();
```

Comment:
No debería ser fontSize / 12. Debería ser 48, como el size of the imported font como ponía el comentario anterior. Es el pixel size que se utiliza al importar la font, por lo que debería mantenerse y modificarse cuando el de  FontImporter se modifique. Cambiaré el 48 a un miembro de FontImporter, pero por ahora, mantén el 48. Para hacer pruebas de cómo se ve la font y si tiene sentido que sea de X tamaño, se puede comparar haciendo pruebas con CSS. 
Entiendo que la transformScale que toque aplicar, mantendrá la relación de aspecto de la fuente.

Reply:
Alright